autoload/yamsong.vim

```diff
@@ -55,6 +55,7 @@ function! yamsong#write() abort
     diffoff
     set buftype=nofile
     bwipe
+    diffoff
```

Comment:
For me, the `diffoff` was not applied to the original buffer, I assume the `diffoff` before the `bwipe` ends up stopping diff mode for only the temporary buffer.